ui/imports/shared/controls/chat/LinkPreviewCard.qml

```diff
@@ -27,11 +31,20 @@ CalloutCard {
     borderWidth: 1
     implicitHeight: 290
     implicitWidth: 305
+    hoverEnabled: true
+    borderColor: hovered || highlight ? Style.current.borderTertiary : Style.current.border
+
+    Behavior on borderColor {
+        ColorAnimation { duration: 200 }
+    }
 
     contentItem: ColumnLayout {
         StatusImage {
             id: bannerImage
             Layout.fillWidth: true
+            Layout.leftMargin: 0.5  // image size isn't pixel perfect..
```

Comment:
Somehow you should `Math.ceil()` the width/height for it to be pixel-aligned; otherwise `size` in QML (width,height) is a float! Not an integer, see https://doc.qt.io/qt-5/qquickitem.html#width-prop

Reply:
I'll do some tests. But I can't test it on low res devices..I think all the devices I have use a pixel ratio of 2

Reply:
Whatever is your final decision could be added as a `d` constant property and reuse it all over the file! :)